Resources/Private/Fusion/Presentation/Slider/Slider.fusion

```diff
@@ -18,37 +19,36 @@ prototype(Neos.Demo:Presentation.Slider) < prototype(Neos.Fusion:Component) {
         gap = 12
     }
 
-    attributes = Neos.Fusion:DataStructure
```

Comment:
Did you notice that with removing this and the following changes for `attributes` you can't add additional attributes to the slider anymore? So this refactoring would change the components behaviour.

Reply:
I wouldn't change the interface in such a PR

Reply:
we actually dont need the `attributes` ourselves (for the integration)

so this was only exposed so possibly other people could use the slider component themselves?

This seems like an odd concept - as for me the neos demo is just a demo without any contract and any expectations any guarantees -> Otherwise our refactor and redesign from 8.2 to 8.3 would have been breaking and illegal.

So i disagree. Imo we are allowed to change it, and in our case it makes the component easier to maintain and migrate to private props.